### ISSUE TITLE
Focus indicator not visible after drilldown

### ISSUE THREAD

**Issue by marvin19** [highcharts/highcharts#22595]
#### Expected behaviour
To be able to use keyboard navigation after drilling down

#### Actual behaviour
When trying to tab into the series after drilling down, the first column does not get focus indicator like it should get, unless you click in the browser first. If you know when the column should be in focus (before the exporting menu) and you press the arrow keys, it is activated. Focus is also activated when tabbing backwards and then forwards again.

#### Live demo with steps to reproduce
[Demo with drilldown](https://jsfiddle.net/maritavindedal/71nhg84p/show)

#### Product version
Highcharts Core

#### Affected browser(s)
Chrome, Safari
